Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -1106,8 +1099,8 @@ public async Task<SystemInfo> GetSystemInfo(CancellationToken cancellationToken)
                 ItemsByNamePath = ApplicationPaths.InternalMetadataPath,
                 InternalMetadataPath = ApplicationPaths.InternalMetadataPath,
                 CachePath = ApplicationPaths.CachePath,
-                OperatingSystem = OperatingSystem.Id.ToString(),
-                OperatingSystemDisplayName = OperatingSystem.Name,
+                OperatingSystem = Environment.OSVersion.VersionString,
+                OperatingSystemDisplayName = RuntimeInformation.OSDescription,
```

Comment:
Keep in mind that this will change from `Linux` to `Unix 5.4.0.1019` etc.

Reply:
I looked through our clients and the only way it was ever used was to display directly to the user, so no breakage should happen.